corehq/apps/export/models/new.py

```diff
@@ -1607,6 +1698,36 @@ def _add_to_group_schema(group_schema, path_start, prop, app_id, app_version):
         return schema
 
 
+class SMSExportDataSchema(ExportDataSchema):
```

Comment:
It's possible that this could be removed with some refactoring. All this provides at the moment is the domain.